locust/dispatch.py

```diff
@@ -16,6 +16,8 @@
 if TYPE_CHECKING:
     from locust.runners import WorkerNode
 
+HEARTBEAT_DEAD = -10
```

Comment:
import from runners instead?

Reply:
I tried that, but was getting error during initialization.
Will try the reverse tomorrow..

Could you help me with any resource on how to setup debugging in distributed mode.. That would be really helpful for me to contribute in a better way.. I am resorting to print statement for debugging(This is my first python project)

Reply:
Sorry, I dont have much to contribute with debugging distributed. You know how to run an individual unit test, right?

Reply:
Yes, initially used to run tox.. later found out the way to run individually.